airbyte-integrations/connectors/source-chargebee/acceptance-test-config.yml

```diff
@@ -1,31 +1,64 @@
 # See [Source Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference)
 # for more information about how to configure these tests
 connector_image: airbyte/source-chargebee:dev
-tests:
+test_strictness_level: high
+acceptance_tests:
   spec:
-    - spec_path: "source_chargebee/spec.json"
+    tests:
+      - spec_path: "source_chargebee/spec.yaml"
   connection:
-    - config_path: "secrets/config.json"
-      status: "succeed"
-    - config_path: "integration_tests/invalid_config.json"
-      status: "failed"
+    tests:
+      - config_path: "secrets/config.json"
+        status: "succeed"
+      - config_path: "integration_tests/invalid_config.json"
+        status: "failed"
   discovery:
-    - config_path: "secrets/config.json"
-      backward_compatibility_tests_config:
-        disable_for_version: "0.1.12"
+    tests:
+      - config_path: "secrets/config.json"
+        backward_compatibility_tests_config:
+          disable_for_version: "0.1.16" # Changes in schemas
```

Comment:
Can you clarify if this is going to break existing customers in production? If yes then how is that being handled?

Reply:
I'll contact cloud support team to communicate with customers (for schema update)